hri_safety_sense/src/JoystickHandler.cpp

```diff
@@ -71,20 +72,26 @@ uint32_t JoystickHandler::handleNewMsg(const VscMsgType &incomingMsg)
 		sensor_msgs::Joy sendLeftMsg;
 
 		sendLeftMsg.header.stamp = ros::Time::now();
-		sendLeftMsg.header.frame_id = "/srcs";
+		sendLeftMsg.header.frame_id = this->frameId;
 
-		sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftX));
-		sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftY));
-		sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftZ));
+		sendLeftMsg.axes.push_back(float(getStickValue(joyMsg->leftX)) /
```

Comment:
Instead of doing all of these casts to float, I think it makes more sense to just have `getStickValue` return a float.

Reply:
Done in fc1be8f